robot/Cumulus/resources/RecurringDonationsPageObject.py

```diff
@@ -217,20 +246,29 @@ def validate_field_values_under_section(self, section=None, **kwargs):
                     label, "contains", value, section
                 )
 
-    def get_next_payment_date_number(self, paynum):
+    def get_next_payment_date_number(self, paynum, format=None):
         """Returns the next payment date from the list of payment schedules taking in the payment number as input
+           The date if formatted to ignore the preceding zeros. But if format is set to false, the date is returned
```

Comment:
can u make it more clear like if format is not set then date is formatted otherwise not

Reply:
or as suggested by safiya, whatever u choose just make it clear in documentation

Reply:
Updated as per Safiya's comment defaulting it to True